private/buf/bufsync/bufsynctest/bufsynctest.go

```diff
@@ -239,7 +241,16 @@ func doManualPushCommit(
 	commitBucket, err := storagegit.NewProvider(repo.Objects()).NewReadBucket(commit.Tree())
 	require.NoError(t, err)
 	moduleBucket := storage.MapReadBucket(commitBucket, storage.MapOnPrefix(moduleDir))
-	fileSet, err := bufcas.NewFileSetForBucket(context.Background(), moduleBucket)
```

Comment:
I guess this one was the reason why the content match wasn't happening on commit-6?

Reply:
Yup